CollAction/Frontend/app/project/Carousel.tsx

```diff
@@ -1,212 +0,0 @@
-import * as React from "react";
```

Comment:
We don't need the carousel anymore at all? @ljwboschker Does that seem right?

Reply:
No, this was the last place it was used. Files can be removed; no use in keeping old stuff around.